.rubocop.yml

```diff
@@ -1,9 +1,22 @@
+AllCops:
+  TargetRubyVersion: 2.3
```

Comment:
Perhaps this could be omitted? The application is configured to use Ruby 2.4.4. RuboCop should look for the `.ruby-version` file and use the version specified by it.

See https://github.com/rubocop-hq/rubocop/blob/master/manual/configuration.md#setting-the-target-ruby-version